lib/internal/event_target.js

```diff
@@ -236,7 +236,7 @@ class Event {
   get returnValue() {
     if (!isEvent(this))
       throw new ERR_INVALID_THIS('Event');
-    return !this.defaultPrevented;
+    return !this.#defaultPrevented;
```

Comment:
> The `returnValue` getter steps are to return `false` if [this](https://webidl.spec.whatwg.org/#this)’s [canceled flag](https://dom.spec.whatwg.org/#canceled-flag) is set; otherwise `true`.

So shouldn't it be:
```suggestion
    return !this.#cancelable;
```

Reply:
I agree with you when I check spec again. Thank you for your help.